node/pkg/driver/node_utils_test.go

```diff
@@ -65,15 +66,15 @@ func TestParseIscsiInitiators(t *testing.T) {
 			filePath := ""
 
 			if tc.file_content != "" {
-				tmpFile, err := ioutil.TempFile(os.TempDir(), "iscis-initiators-")
+				tmpFile, err := ioutil.TempFile(os.TempDir(), "nvme-")
 				fmt.Println(tmpFile)
 				if err != nil {
 					t.Fatalf("Cannot create temporary file : %v", err)
 				}
 
 				defer func() {
 					os.Remove(tmpFile.Name())
-					driver.IscsiFullPath = "/host/etc/iscsi/initiatorname.iscsi"
+					driver.IscsiFullPath = "/host/etc/nvme/hostnqn"
```

Comment:
iscsi? really?
please go over this PR diff as well. we are lacking reviewers

Reply:
done